include/boost/fusion/tuple/tuple.hpp

```diff
@@ -13,7 +13,80 @@
 ///////////////////////////////////////////////////////////////////////////////
 // With no variadics, we will use the C++03 version
 ///////////////////////////////////////////////////////////////////////////////
+#if !defined(BOOST_FUSION_HAS_VARIADIC_TUPLE)
 # include <boost/fusion/tuple/detail/tuple.hpp>
+#else
 
+///////////////////////////////////////////////////////////////////////////////
+// C++11 interface
+///////////////////////////////////////////////////////////////////////////////
+#include <boost/fusion/container/vector/vector.hpp>
+#include <boost/fusion/sequence/intrinsic/size.hpp>
+#include <boost/fusion/sequence/intrinsic/value_at.hpp>
+#include <boost/fusion/sequence/intrinsic/at.hpp>
+#include <boost/fusion/sequence/comparison.hpp>
+#include <boost/fusion/sequence/io.hpp>
+#include <utility>
+
+namespace boost { namespace fusion
+{
+    template <typename ...T>
+    struct tuple : vector<T...>
+    {
```

Comment:
Alias templates are not always supported when variadics are, but even if they were these types are documented as different things, and making them the same could/would be problematic for partial specializations.

Furthermore, there's the question of how much should `fusion::tuple` try to be like `std::tuple`, specially considering `fusion::tuple` started as a TR1 implementation of `tuple`. Otherwise, there might not be much sense in keeping this around.